pages/gallery/modal.js

```diff
@@ -0,0 +1,285 @@
+                          hide-default-footer
+                        ></v-data-table>
+                      </div>
+                    </template>
+                  </template>
+                  <div v-if="item === items[1]" class="double_table">
+                    <template v-for="i in authors">
+                      <div style="padding: 15px">
+                        <h2 style="text-transform: capitalize;">{{ i }}</h2>
+                        <v-data-table
+                          dense
+                          :headers="getHeaders(i)"
+                          :items="getItems(i)"
+                          class="elevation-1"
+                          style="margin-top: 10px"
+                          hide-default-footer
+                        ></v-data-table>
+                      </div>
+                    </template>
+                  </div>
+                  <div v-if="item === items[2]">
+                    {{ $root.lang().global.nyi }}
+                  </div>
+                  <div v-if="item === items[3]">
+                    {{ $root.lang().global.nyi }}
+                  </div>
+                </v-tab-item>
+              </v-tabs-items>
+            </div>
+          </div>
+        </template>
+      </v-card>
+
+    </v-dialog>
+  `,
+  props: {
+    opened: {
+      type: Boolean,
+      required: true
+    },
+    closeModal: {
+      type: Function,
+      required: true
+    },
+    textureID: {
+      type: String,
+      required: true
+    },
+    textureObj: {
+      type: Object,
+      required: true
+    },
+    contributors: {
+      type: Object,
+      required: true
+    }
+  },
+  data() {
+    return {
+      resolutions: ['16x', ...settings.resolutions],
+      tab: null,
+      items: [
+        this.$root.lang().gallery.modal.items.information,
+        this.$root.lang().gallery.modal.items.authors,
+        this.$root.lang().gallery.modal.items.animated,
+        this.$root.lang().gallery.modal.items.model
+      ],
+      infos: ["texture", "uses", "paths"],
+      infosText: {
+        texture: this.$root.lang().gallery.modal.infos.texture,
+        uses: this.$root.lang().gallery.modal.infos.uses,
+        paths: this.$root.lang().gallery.modal.infos.paths,
+      },
+      authors: settings.resolutions
+    }
+  },
+  methods: {
+    discordIDtoName(d) {
+      return this.contributors[d].username || this.$root.lang().gallery.error_message.user_not_found
+    },
+    timestampToDate(t) {
+      const a = new Date(t)
+      const months = [this.$root.lang().global.months.jan, this.$root.lang().global.months.feb, this.$root.lang().global.months.mar, this.$root.lang().global.months.apr, this.$root.lang().global.months.may, this.$root.lang().global.months.jun, this.$root.lang().global.months.jul, this.$root.lang().global.months.aug, this.$root.lang().global.months.sep, this.$root.lang().global.months.oct, this.$root.lang().global.months.nov, this.$root.lang().global.months.dec]
```

Comment:
I have seen this before, maybe try to look at what "Vue mixins" are

Reply:
Where is the issue?